### ISSUE TITLE
Don't prevent block_break from minecraft:falling_block entities.

### ISSUE THREAD

**Issue by me4502** [MinecraftPortCentral/GriefPrevention#539]
Falling Block entities, when spawned, take the type of the block where they are spawned, and then remove the block. Currently GriefPrevention prevents the falling block entity from removing this block, creating a dupe bug.

This affects anyone who uses Command Blocks for this kind of thing, or any other plugins that spawn falling block entities.

Here's a report of a ladder falling block being spawned, https://gist.github.com/anonymous/6866b793f90d5a13cc09af14b1f230c9

**Comment by KaneHart** [comment]
I don't know if this the same issue or not but I might as well tell my story and someone can decide from there.

I had spawn protection on and GP admin claim at spawn. I started fresh ATM3 World and before i logged on my server performance was slowly getting worse and worse as time went on. The Entities count was climbing I had no clue what was going on.

In the end I found out was gravel falling to the ground and there was a torch like item from a mod breaking the gravel. Either way the gravel was duping from the air down to the cavern and causing 20,000+ entities to land on the ground.

Now my GP entity collision was disabled but if it was enabled I was of locked my box up 100 times sooner.
